android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -132,7 +157,51 @@ protected void triggerLayoutInflation() {
 
         TextView assetTitleText = findViewById(R.id.asset_title_text);
         assetTitleText.setText(mAssetName);
-        if (!mAssetLogo.isEmpty()) {
+        mBtnBuy = findViewById(R.id.btn_buy);
+        Button btnSend = findViewById(R.id.btn_send);
+        mBtnSwap = findViewById(R.id.btn_swap);
+
+        if (mCoinMarket) {
+            TextView informationLabel = findViewById(R.id.information);
+            CardView coinMarketInfo = findViewById(R.id.card_view_coin_market_info);
```

Comment:
nit: personally I would prefer to use the term `MarketCoin` for readability when possible, than the core term `CoinMarket`.

Reply:
No problem, I'm changing it.